src/widgets/pixbuf_loader.rs

```diff
@@ -18,16 +18,12 @@ pub struct PixbufLoader {
 impl PixbufLoader {
     /// Creates a new pixbuf loader object.
     ///
-    /// # Failures
-    /// Returns `None` if the pixbuf loader cannot be created.
-    pub fn new() -> Option<PixbufLoader> {
+    /// # Panics
+    /// Fails if the pixbuf loader cannot be retrieved. 
+    pub fn new() -> PixbufLoader {
```

Comment:
We discussed about returning objects without Option wrapping somewhere but we didn't started it yet. Could it possible for now to keep returning `Option<PixbufLoader>` please ?